CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to Pulumi
+
+Do you want to hack on Pulumi?  Awesome!  We are so happy to have you.
+
+Please refer to the [main Pulumi repo](https://github.com/pulumi/pulumi/)'s [CONTRIBUTING.md file](
+https://github.com/pulumi/pulumi/blob/master/CONTRIBUTING.md) for details on how to do so.
+
+## Committing Generated Code
```

Comment:
I would remove this section.  We don't have any generated code in the AWSX package, unlike AWS.